src/main/java/org/codehaus/mojo/versions/AbstractVersionsUpdaterMojo.java

```diff
@@ -417,10 +417,16 @@ protected abstract void update( ModifiedPomXMLEventReader pom )
      * @return <code>true</code> if the update should be applied.
      * @since 1.0-alpha-1
      */
-    protected boolean shouldApplyUpdate( Artifact artifact, String currentVersion, ArtifactVersion updateVersion )
+    protected boolean shouldApplyUpdate( Artifact artifact, String currentVersion, ArtifactVersion updateVersion, Boolean forceUpdate )
```

Comment:
please use `boolean` and keep previous method marking it as deprecated and add this new method

Reply:
Done as suggested. Please review again.